src/Parallel-Solvers/Parallel-Explicit/example_simple.yaml

```diff
@@ -6,11 +6,22 @@ dynamic_options:
   dt_start: 1.e-5
   cycle_stop: 2000000
 
-mesh_generation_options:
-  type: Box
-  origin: [0, 0, 0]
-  length: [1.2, 1.2, 1.2]
-  num_elems: [32, 32, 32]
+#mesh_generation_options:
+#  type: Box
+#  origin: [0, 0, 0]
+#  length: [10, 10, 10]
+#  num_elems: [12, 12, 12]
+
+input_options:
+  mesh_file_format: vtk
+  mesh_file_name: /Users/shankins/Documents/FY24/Github/XcodeFierro/Fierro/python/temp_files/global_mesh/mesh.vtk
```

Comment:
I recommend changing this before you push it to main to be something like
mesh_file_name: <path/to/mesh/file>
That way people will know that this needs to be manually modified.

Reply:
This file has been reverted to match what is currently on the main branch.